app/components/patientcard/patientcard.js

```diff
@@ -220,7 +220,7 @@ var PatientCard = React.createClass({
   },
 
   renderUploadOverlay: function() {
-    return <UploadLaunchOverlay overlayClickHandler={()=>{this.setState({showUploadOverlay: false})}}/>
+    return <UploadLaunchOverlay modalDismissHandler={()=>{this.setState({showUploadOverlay: false})}}/>
```

Comment:
After renaming the `overlayClickHandler` method  to `modalDismissHandler`, I'd suggest cleaning this duplicated up to use the instance method instead of re-creating the handler:
```
return <UploadLaunchOverlay modalDismissHandler={this.modalDismissHandler}/>
```